src/components/Entry.vue

```diff
@@ -110,6 +110,9 @@ export default {
       isLoading: false,
       invalidInputMessage: null, // message to show when the input being entered does not match the ID format,
       userType: "", // differentiates between different kinds of users
+      loadingSpinner: new URL('../assets/images/loading_spinner.svg', import.meta.url).href,
+      deleteSvg: new URL('../assets/images/remove_circle.svg', import.meta.url).href,
+      addSvg: new URL('../assets/images/add_circle.svg', import.meta.url).href
```

Comment:
Also, if possible, try to maintain consistency in naming!
Like one svg is named `deleteSvg` but another one is named `loadingSpinner`.
If you're adding `svg` at the end of the name, make sure to do that for all the svgs